views/job-roles.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <!-- font icons -->
+    <link rel="stylesheet" href="public/vendors/themify-icons/css/themify-icons.css">
+
+    <!-- Bootstrap + Creative Studio main styles -->
+    <link rel="stylesheet" href="public/creative-studio.css">
+
+    <meta charset = "UTF-8">
+    <title>Your Kainos story starts here</title>
+    <link type="image/x-icon" href="https://cdn.phenompeople.com/CareerConnectResources/pp/KAINGB/en_gb/desktop/assets/images/favicon.ico" rel="icon" data-ph-id="ph-page-element-page24-SpBhhG">
+    <link rel="preload" as="image" importance="high" href="https://cdn.phenompeople.com/CareerConnectResources/pp/KAINGB/images/cropChild-SoQGk-1653562074041.png" id="ph_image_preload">
+
+    <!--Mobile optimisation-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+
+</head>
+<body data-spy="scroll" data-target=".navbar" data-offset="40" id="home">
+    
+    <!-- Page Navigation -->
+    <nav class="navbar custom-navbar navbar-expand-lg navbar-light" data-spy="affix" data-offset-top="20">
```

Comment:
if you moved the header, footer and nav bar out to their own html files you could use nunjucks to include them on all pages in 1 line rather than repeating all of this code